packages/hashi/src/kolibri.js

```diff
@@ -94,6 +94,9 @@ export default class Kolibri extends BaseShim {
        * @param {string[]} options.ids - an array of ids to filter by
        * @param {number} [options.page=1] - which page to return from the result set
        * @param {number} [options.pageSize=50] - the page size for pagination
+       * @param {string} options.random - set to a random string to get the result randomly ordered
```

Comment:
From the user of the API, it would be better to say "randomize: true" than passing a random string. Or is this option the random seed?

Reply:
It's a random "seed". The API is caching every request, so we can't use a boolean here because in that case we'll get the same ordering on every request with the same parameters.